templates/main.template

```diff
@@ -271,36 +371,15 @@ Resources:
             - s3-us-gov-west-1
             - s3
       TimeoutInMinutes: 20
-  LoggingTemplate:
-    Type: AWS::CloudFormation::Stack
-    Properties:
-      TemplateURL:
-        !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-compliance-common/templates/logging.template
-        - QSS3Region:
-            !If
-            - GovCloudCondition
-            - s3-us-gov-west-1
-            - s3
-      TimeoutInMinutes: 20
-      Parameters:
-        pNotifyEmail: !Ref pNotifyEmail
-        pSupportsGlacier:
-          !FindInMap
-          - RegionServiceSupport
-          - !Ref AWS::Region
-          - Glacier
   ProductionVpcTemplate:
     Type: AWS::CloudFormation::Stack
     Properties:
-      TemplateURL:
-        !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-compliance-common/templates/vpc-production.template
-        - QSS3Region:
-            !If
-            - GovCloudCondition
-            - s3-us-gov-west-1
-            - s3
+      TemplateURL: https://cloudformationlctemplates.s3.amazonaws.com/vpc-production.template_LCQS
```

Comment:
This will need to be pointed to the standard URL format.  I'd like to better understand what updates you made.

Reply:
Really just clarifying security group descriptions and restricting the security group from mgmt VPC to Prod VPC from only the bastion IP

Reply:
rSecurityGroupMgmtBastion:
    Type: AWS::EC2::SecurityGroup
    DependsOn:
      - rMgmtBastionInstance
      - rENIProductionBastion
    Properties:
      GroupDescription: Allow Bastion internal IP from managementVPC into ProductionVPC via SSH
      VpcId: !Ref pProductionVPC
      SecurityGroupIngress:
      - IpProtocol: tcp
        FromPort: 22
        ToPort: 22
        CidrIp: !Join [ '', [ !GetAtt rENIProductionBastion.PrimaryPrivateIpAddress, '/32' ] ]
      Tags:
      - Key: Name
        Value: sg-ssh-access-from-management-vpc